src/components/Confirmation/Confirmation.styles.js

```diff
@@ -2,6 +2,10 @@ import { css } from '@emotion/core';
 import { Space } from 'lib/theme';
 
 export const styles = {
+  check: css({
+    marginLeft: Space.S10,
```

Comment:
I think `title` has marginRight: 10 in this file, can you double check we aren't adding margin twice?

Reply:
🤘taking it off `.title` (which as getting applied to the container)